src/main/java/com/browserstack/automate/ci/common/proxysettings/JenkinsProxySettings.java

```diff
@@ -0,0 +1,64 @@
+package com.browserstack.automate.ci.common.proxysettings;
+
+import hudson.ProxyConfiguration;
+import jenkins.model.Jenkins;
+import okhttp3.Credentials;
+
+import java.net.InetSocketAddress;
+import java.net.Proxy;
+
+public class JenkinsProxySettings {
```

Comment:
Just a suggestion from a coding style point of view,
it's better to have line separations between function definitions. And also inside function definitions where the logical part, or "group of steps" separate out...

Reply:
Resolving.